src/features/network-modifications/common/reactiveLimits/reactiveCapabilityCurve/reactiveCapability.utils.ts

```
@@ -6,15 +6,14 @@
 import { FieldValues, UseFormSetValue } from 'react-hook-form';
-import { ReactiveCapabilityCurve, ReactiveCapabilityCurvePoints } from '../reactiveLimits.type';
+import { ReactiveCapabilityCurve } from '../reactiveLimits.type';
 import { FieldConstants, toNumber, validateValueIsANumber } from '../../../../../utils';
-import {
-    GeneratorCreationDialogSchemaForm,
-    GeneratorDialogSchemaBaseForm,
-    GeneratorModificationDialogSchemaForm,
-} from '../../../generator/generatorDialog.type';
+import type { GeneratorCreationFormData } from '../../../generator/creation/generatorCreation.utils';
 import { GeneratorCreationDto } from '../../../generator/creation/generatorCreation.types';
 import { GeneratorModificationDto } from '../../../generator/modification/generatorModification.types';
```

ESLint

[error] Dependency cycle via "../../common:8=>./reactiveLimits:15=>./reactiveCapabilityCurve:8"

(import-x/no-cycle)